### DISCUSSION TITLE
Can I see previous version once edited?

### DISCUSSION THREAD

**Discussion by HubKing** [usememos/memos#2277] [Q&A] [answered]
I mean, if you edit your post on GitHub, you can see the previous version by clicking the history button.
Does Memos have similar feature?

![image](https://github.com/usememos/memos/assets/10900489/ab45db11-0d4f-4aa9-abef-5540cf6305fb)

**Comment by diomed** [comment, marked as answer]
no you cant. 
unless you maybe do some git tracking.
that would require some knowledge and skils.

but no, in default behavior that's not possible.

**Reply by almereyda** [reply]
Good hint that with file system storage git can be used.

Similarly on S3 storage, bucket versioning could potentially also be used.

Also see:

- #798
